events/wc_events/wc_uther_events.txt

```diff
@@ -0,0 +1,64 @@
+﻿namespace = uther_events
+
+# Death of Terenas
+uther_events.001 = {
```

Comment:
Make this event more generic in case it's not Arthas (see my comment on the on_action)

Reply:
same comment from below